war/src/main/less/base/layout-commons.less

```diff
@@ -0,0 +1,140 @@
+/*
+ * The MIT License
+ *
+ * Copyright (c) 2016, CloudBees, Inc.
```

Comment:
Same with this one. Where is it sources from that needs it to be specifically licensed from cloudbees

Reply:
Removed